### ISSUE TITLE
DateTime not initialized to MinValue

### ISSUE THREAD

**Issue by fabriciomurta** [bridgedotnet/Bridge#3734]
A System.DateTime object is not initialized to its default minimum value by default, and the current time zone may affect the results displayed.

### Steps To Reproduce

https://dev.deck.net/3c7cda50410ec2e1abf5240d6f52aa0b

```csharp
public class Program
{
    static DateTime A;	// Not initialized to MinValue
    static DateTime B = DateTime.MinValue;
    
    public static void Main()
    {
        Console.WriteLine(A);
        Console.WriteLine(B);
        Console.WriteLine(A.Ticks);
        Console.WriteLine(B.Ticks);

        if (A == DateTime.MinValue)
            Console.WriteLine("A is min value");
            
        if (B == DateTime.MinValue)
            Console.WriteLine("B is min value");
    }
}
```

### Expected Result

- [Native .NET test case in dotnet fiddle](https://dotnetfiddle.net/Tw7OLN)
```
Success: DateTime A is '1/1/0001 12:00:00 AM'.
Success: DateTime B is '1/1/0001 12:00:00 AM'.
Success: DateTime A has 0 ticks.
Success: DateTime B has 0 ticks.
Success: A is min value.
Success: B is min value.
Statistics:
- Passing: 6 (100%)
- Failing: 0 (0%)
- Total: 6
```

### Actual Result

- [Deck mimicking above native code](https://deck.net/c8a35e2fd51d8f4dd23d71d39085b938)
```
Failure: DateTime A is '1/1/0001 12:00:00 AM'.
Failure: DateTime B is '1/1/0001 12:00:00 AM'.
Failure: DateTime A has 0 ticks.
Success: DateTime B has 0 ticks.
Failure: A is min value.
Success: B is min value.
Statistics:
- Passing: 2 (33%)
- Failing: 4 (67%)
- Total: 6
```

### See Also

- [DateTime not initialized to MinValue](https://forums.bridge.net/forum/bridge-net-pro/bugs/5922) thread by [dosh](https://forums.bridge.net/member/449).

**Comment by geoffreymcgill** [comment]
The reason for the first two failures is just due to the default DateTime FormatSpecifier this is used based on the current CultureInfo. Deck using InvariantCulture and DotNetFiddle (and probably local) is using `en-US`. 

```
Failure: DateTime A is '1/1/0001 12:00:00 AM'.
Failure: DateTime B is '1/1/0001 12:00:00 AM'.
```

If both environments used the same CultureInfo settings, the default string formats would match.

**Comment by geoffreymcgill** [comment]
This scenario has been fixed. After another review and final testing, the fix will be included in the upcoming Bridge 17.8.0 release.

You can confirm with the following dev.deck.net sample:

https://dev.deck.net/3c7cda50410ec2e1abf5240d6f52aa0b

**Comment by fabriciomurta** [comment]
A full test over all time zones have been undegone and all looks good, so the fix for this will be available next Bridge release!
